CONTRIBUTING.md

```diff
@@ -38,9 +46,9 @@ that with the npm `-g` flag.
 npm install -g gulp-cli
 ```
 
-#### Install all the things
+#### Install all the NPM packages
 
-With all of those in place, simply run:
+With all of those in place, run:
 
 ```
 npm install
```

Comment:
It's worth mentioning we use version `8.9.4`.
If one does not have that (that's old) then `nodenv` should be used.